dspace-api/src/main/java/org/dspace/authority/AuthorityValue.java

```diff
@@ -468,16 +468,14 @@ public void updateConceptFromAuthorityValue(Concept concept) throws SQLException
                 for(String value : otherMetadata.get(name))
                 {
                     // Add it to the list
-                    concept.addMetadata(schema,element,qualifier,"",value,null,-1);
+                    concept.addMetadata(context, schema,element,qualifier,"",value,null,-1);
```

Comment:
Agreed. I don't know why it was put in there like that in the first place. -1 is equal to `Choices.CF_UNSET`.